src/hooks/useLoginForm/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {
+  emailValidator,
+  passwordValidator
+} from 'validators';
+
+const useLoginForm = () => {
+  const [inputs, setInputs] = React.useState({
+    email: '',
+    password: ''
+  });
+  const handleSubmit = async (event) => {
+    if (event) {
+      event.preventDefault();
+      if (!emailValidator(inputs.email)) {
+        console.log('incorrect email entered')
```

Comment:
Rather than console logging, we should send some kind of notification to the user either through an error underneath the component or some toast notification telling them that their email is incorrect/not found. They shouldn't need to open the console in order to find this information

Reply:
Will implement user facing validation errors in the future